examples/platform/name_a_user.py

```diff
@@ -0,0 +1,44 @@
+#!/usr/bin/env python
+# *******************************************************
+# Copyright (c) VMware, Inc. 2020-2021. All Rights Reserved.
+# SPDX-License-Identifier: MIT
+# *******************************************************
+# *
+# * DISCLAIMER. THIS PROGRAM IS PROVIDED TO YOU "AS IS" WITHOUT
+# * WARRANTIES OR CONDITIONS OF ANY KIND, WHETHER ORAL OR WRITTEN,
+# * EXPRESS OR IMPLIED. THE AUTHOR SPECIFICALLY DISCLAIMS ANY IMPLIED
+# * WARRANTIES OR CONDITIONS OF MERCHANTABILITY, SATISFACTORY QUALITY,
+# * NON-INFRINGEMENT AND FITNESS FOR A PARTICULAR PURPOSE.
+
+"""Example script which finds a user without a name and gives them one."""
```

Comment:
Instead of finding a random user which is not a practical customer usecase this should update an existing user by email either their name or phone number. The code is also really dense and compressed which makes reading and comprehending the script a challenge for newer python developers.

Reply:
I may just dump this whole script as it doesn't add much useful to the examples. Would that be OK?

Reply:
That works too

Reply:
It's been sent to the bit bucket.